cognite_toolkit/custom_modules/my_example_module/README.md

```diff
@@ -1,3 +0,0 @@
-# README.md
-
-This is an example of a custom module. It contains two TimeSeries with a dataset.
```

Comment:
Any reason why you deleted this module? I guess that has nothing to do with the core change of adding Workflow support?

Reply:
Nope, got a bit eager. Reverted.